### ISSUE TITLE
Re-title wiki.civicrm.org so Docs site is preferred

### ISSUE THREAD

**Issue by xurizaemon** [civicrm/civicrm-docs#62]
Currently searches for "civicrm docs" have the wiki at the top. By now I think we should be preferring to land people on docs.civicrm.org and linking to "legacy" docs in the wiki when necessary.

Not sure what is required to re-title wiki.civicrm.org "CiviCRM Wiki" or "Legacy Documentation" to denote this, so opening an issue to propose it.

![screen shot 2017-04-22 at 11 51 08 pm](https://cloud.githubusercontent.com/assets/105608/25304220/f191eb42-27b6-11e7-8990-86498772896b.png)

**Comment by seanmadsen** [comment]
Agreed. I think that "CiviCRM Documentation" text is coming from (1) the page title (easy to edit), and (2) the *space* title as shown [here](https://wiki.civicrm.org/confluence/spacedirectory/view.action) (which I don't think I have perms to edit). I'll see if I can get my perms elevated so that I can work on this.

**Comment by mlutfy** [comment]
Josh OK'ed it, and someone has already changed the title to "CiviCRM Wiki".
